helper/keysutil/lock_manager.go

```diff
@@ -386,7 +409,7 @@ func (lm *LockManager) DeletePolicy(ctx context.Context, storage logical.Storage
 	lock.Lock()
 	defer lock.Unlock()
 
-	pRaw, ok := lm.cache.Load(name)
+	pRaw, ok = lm.cache.Load(name)
```

Comment:
Do we want to check the useCache value before this call?

Reply:
Yes. Yes I do.